CMakeLists.txt

```diff
@@ -56,7 +56,7 @@ find_package(PCRE2 REQUIRED)
 if(USE_BUILTIN_SENTENCEPIECE)
   list(APPEND SLIMT_PUBLIC_LIBS SentencePiece::SentencePiece)
 else(USE_BUILTIN_SENTENCEPIECE)
-  list(APPEND SLIMT_PRIVATE_LIBS SentencePiece::SentencePiece)
+  list(APPEND SLIMT_PUBLIC_LIBS SentencePiece::SentencePiece)
```

Comment:
As in if and else we have the same code for SentencePiece::SentencePiece
we can change code as

```
list(APPEND SLIMT_PUBLIC_LIBS SentencePiece::SentencePiece)
if (NOT USE_BUILTIN_SENTENCEPIECE)
   list(APPEND SLIMT_PRIVATE_LIBS SentencePiece::Protobuf)
endif()
```

Reply:
https://github.com/jerinphilip/slimt/pull/26/commits/630bd99ce2d055bbf5978dfd27d75f549b92f618